Codeinwp/feedzy-rss-feeds
# fix: feed image when using chained actions

### Summary


In https://github.com/Codeinwp/feedzy-rss-feeds/pull/944 we allowed the custom tags to work with featured images. That condition established a default image for the chained action tags.

I fixed and re-write the condition to make its purpose more clear. Also corrected the serialization for image chained actions.

> [!NOTE]
> The users affected by this issue were those using empty chained actions for the featured image.

### Will affect visual aspect of the product

NO

### Screenshots 

### Test instructions


- Create a new feed and add via `Insert Tag` the tag `Item Image`
- Also install the Pro version and check the image generation with OpenAI via chained actions.

## Check before Pull Request is ready:



Closes https://github.com/Codeinwp/feedzy-rss-feeds/issues/945
